packages/astro/src/core/build/generate.ts

```diff
@@ -284,7 +284,7 @@ async function getPathsForRoute(
 		const label = staticPaths.length === 1 ? 'page' : 'pages';
 		logger.debug(
 			'build',
-			`├── ${bold(green('✔'))} ${route.component} → ${magenta(`[${staticPaths.length} ${label}]`)}`
+			`├── ${bold(green('√'))} ${route.component} → ${magenta(`[${staticPaths.length} ${label}]`)}`
```

Comment:
why did this change?

Reply:
I didn't, Biome did (bug)

Reply:
Ah ok, can you save it without the formatter in this case then?